frontend/asset_mapper.rst

```diff
@@ -289,17 +289,25 @@ Preloading and Initializing "app.js"
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In addition to the importmap, the ``{{ importmap() }}`` Twig function also renders
-an `ES module shim`_ and a few other things, like a set of "preloads":
+an `ES module shim`_. You can disable this in your ``config/packages/asset_mapper.yaml``
+configuration file:
 
-.. code-block:: html
+.. code-block:: yaml
```

Comment:
we should have the configuration block for all supported formats, like elsewhere

Reply:
Sorry, I don't know the other formats' syntax :-( But starting with YAML is certainly better than nothing...